app/resources/css-default.template

```diff
@@ -194,6 +196,23 @@ input {
   font-size: 13px;
   margin-bottom: .5em;
   max-width: 100%;
+  border: 0;
+  outline: 0;
+  border-bottom: 1px solid rgb(169, 169, 169);
```

Comment:
Can you be consistent with the color code format? Other parts uses #xxxxxx format, so it would be nice to use it here too. Same for below.